CHANGES.md

```diff
@@ -1,5 +1,17 @@
 ### Unreleased
 
+#### Improve and simplify LP lexer (2021-12-02)
+
+- allow nested comments (fix #710)
+- generate only one kind of identifiers
+- replace everywhere `%S` by `\"%s\"`
```

Comment:
Did `%S` escape things that shouldn't have been?

Reply:
%S converts a string argument to OCaml syntax (double quotes, escapes). There is no reason for doing this in LP.